### PR TITLE
DevKit updates for 2.x branch

### PR DESCRIPTION
**Pull request by SonataCI**

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -22,7 +28,6 @@
 
     <php>
         <ini name="precision" value="8" />
-        <env name="KERNEL_CLASS" value="Sonata\DoctrinePHPCRAdminBundle\Tests\Fixtures\App\Kernel" />
```
Comment (jordisala1991) [inline phpunit.xml.dist:25]: Either this env var is defined in custom bootstrap somehow , added to dev-kit or this file cannot be generic

Comment (VincentLanglet) [reply phpunit.xml.dist:25]: I tried 
```
public static function getKernelClass(): string
    {
        return Kernel::class;
    }
```

Seems like it doesn't work...

Comment (VincentLanglet) [reply phpunit.xml.dist:25]: @sonata-project/contributors Any idea ?

**Review by OskarStark — DISMISSED** [review]

**Review by VincentLanglet — CHANGES_REQUESTED** [review]
Should not be merged like this

**Review by VincentLanglet — APPROVED** [review]
Finally

**Review by jordisala1991 — APPROVED** [review]

**Pull request merged**